kliff/transforms/property_transforms.py

```diff
@@ -0,0 +1,197 @@
+        self.mean = original_property_values.mean()
+        self.std = original_property_values.std()
+        for configuration in configuration_list:
+            property_to_transform = getattr(configuration, self.property_key)
+            property_to_transform -= self.mean
+            property_to_transform /= self.std
+            setattr(configuration, self.property_key, property_to_transform)
+
+    def inverse(self, dataset: Union[List["Configuration"], "Dataset"]):
+        configuration_list = self.get_configuration_list(dataset)
+        n_configs = len(configuration_list)
+        for configuration in configuration_list:
+            property_to_transform = getattr(configuration, self.property_key)
+            property_to_transform *= self.std
+            property_to_transform += self.mean
+            setattr(configuration, self.property_key, property_to_transform)
+
+
+class RMSNormalizePropertyTransform(PropertyTransform):
+    """
+    Normalize the property of a configuration to zero mean and unit variance,
+    using the root mean square of the property.
+    """
+
+    def __init__(self, property_key="forces", keep_original=False):
+        super().__init__(property_key, keep_original)
+        self.original_property_value_map = None
+        self.property_key = property_key
+        self._keep_original = keep_original
+        self.rms_mean = 0.0
+
+    def transform(self, dataset: Union[List["Configuration"], "Dataset"]):
+        configuration_list = self.get_configuration_list(dataset)
+        n_configs = len(configuration_list)
+        original_property_values = list(
+            map(lambda config: getattr(config, self.property_key), configuration_list)
+        )
+        original_property_values = np.vstack(original_property_values)
+        if self.keep_original:
+            self.original_property_value_map = original_property_values
+        self.rms_mean = np.sqrt(np.mean(np.square(original_property_values)))
+        for configuration in configuration_list:
+            property_to_transform = getattr(configuration, self.property_key)
+            property_to_transform /= self.rms_mean
+            setattr(configuration, self.property_key, property_to_transform)
+
+    def inverse(self, dataset: Union[List["Configuration"], "Dataset"]):
+        configuration_list = self.get_configuration_list(dataset)
+        n_configs = len(configuration_list)
+        for configuration in configuration_list:
+            property_to_transform = getattr(configuration, self.property_key)
+            property_to_transform *= self.rms_mean
+            setattr(configuration, self.property_key, property_to_transform)
+
+
+class RMSMagnitudeNormalizePropertyTransform(PropertyTransform):
+    """
+    Normalize the property of a configuration to zero mean and unit variance,
+    using the root mean square of the magnitude of the property. This method is
+    useful for normalizing forces.
```

Comment:
More docs need to explain this and `RMSNormalizePropertyTransform`. Ideally, providing the mathematical formula on how it is computed?